tests/views_integration/README.md

```diff
@@ -316,10 +316,155 @@ This second test example shows what is, probably, the main feature of data-drive
 * we manipulate data as the full View, in production, would do
 
 ### Snapshot testing
```

Comment:
See comment about possible title change below.

Reply:
@juanfra @paulmskim I think we're mixing **what we test** (View data, View HTML, Partial HTML) with **how** we test it (db queries, snapshot testing, HTML structure lookup).
This section should be renamed to "View (HTML) testing".

Snapshot testing is the testing technique used for testing entire Views, so it makes sense to explain it here.
Please add a sub-section dedicated to snapshot testing and move the presentation of the technique there; this should serve as an high level introduction for it, that could be referenced in the "View (HTML) Testing" and "Component (HTML) Testing" sections.